apps/admin-portal/src/components/users/user-profile.tsx

```diff
@@ -49,10 +55,14 @@ export const UserProfile: FunctionComponent<ProfileProps> = (props: ProfileProps
     const { onAlertFired, user, handleUserUpdate } = props;
     const { t } = useTranslation();
 
-    const [profileInfo, setProfileInfo] = useState(new Map<string, string>());
-    const [profileSchema, setProfileSchema] = useState<ProfileSchema[]>();
-    const profileDetails: AuthStateInterface = useSelector((state: AppState) => state.authenticationInformation);
-    const [urlSchema, setUrlSchema] = useState<ProfileSchema>();
+    const [ profileInfo, setProfileInfo ] = useState(new Map<string, string>());
+    const [ profileSchema, setProfileSchema ] = useState<ProfileSchema[]>();
+    const profileDetails: AuthStateInterface = useSelector((state: AppState) =>
+        state.authenticationInformation);
+    const [ urlSchema, setUrlSchema ] = useState<ProfileSchema>();
+
```

Comment:
Shall we remove this empty space or add a comment if you want to separate them

Reply:
I separated them cos they were related to confirmation modal. Should we add a comment though?